sdk/instrumentation/opentelemetry-instrumentation-azure-sdk/src/configuration.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export const SDK_VERSION: string = "1.0.0-beta.2";
+
+/**
+ * @internal
+ *
+ * Keys of known environment variables we look up.
+ */
+export type KnownEnvironmentKey = "AZURE_HTTP_TRACING_DISABLED" | "AZURE_TRACING_DISABLED";
+
+export const environment = new Map<KnownEnvironmentKey, string | undefined>();
+
+if (typeof process !== "undefined" && process.env) {
+  environment.set("AZURE_HTTP_TRACING_DISABLED", process.env.AZURE_HTTP_TRACING_DISABLED);
+  environment.set("AZURE_TRACING_DISABLED", process.env.AZURE_TRACING_DISABLED);
+}
+
+/**
+ * Converts an environment variable to Boolean.
+ * the strings "false" and "0" are treated as falsy values.
+ *
+ * @internal
+ */
+export function envVarToBoolean(key: KnownEnvironmentKey): boolean {
```

Comment:
I wonder if this small utility function should live in @azure/core-util instead so that we can have a consistent interpretation of boolean environment variables. What do you all think?

Reply:
I'm curious, if we move this utility to core-util, would `environment` be global on @azure/core-util, or would this be updated to take the value instead of the key?

Reply:
My first reaction is I think I would only expose the conversion part, so like

```ts
export function toBoolean(rawValue?: string): boolean {
  const value = (rawValue ?? "").toLowerCase();
  return value !== "false" && value !== "0" && Boolean(value);
}
```

Or something like that

Reply:
I could have sworn we had similar logic to what .NET is doing somewhere... but I can't find it. I like the idea of having it in a common place, ideally of the shape

```ts
isEnvBooleanTrue(environmentVariableName: string): boolean
```

Reply:
Thanks, I created #20835 for this